### PR TITLE
[CARBONDATA-3840] Mark features as experimental

### PR DESCRIPTION
**Pull request by kunal642**
### Why is this PR needed?
 Mark features as experimental because they are subject to change in future.
 
 ### What changes were proposed in this PR?
 Mark features as experimental because they are subject to change in future.
    
 ### Does this PR introduce any user interface change?
 - No

 ### Is any new testcase added?
 - No

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,6 +69,14 @@ CarbonData is built using Apache Maven, to [build CarbonData](https://github.com
 * [Carbon as Spark's Datasource](https://github.com/apache/carbondata/blob/master/docs/carbon-as-spark-datasource-guide.md) 
 * [FAQs](https://github.com/apache/carbondata/blob/master/docs/faq.md) 
 
+## Experimental Features
+
+Some features are marked as experimental because the syntax/implementation might change in the future.
+1. Hybrid format table using Add Segment.
+2. Accelerating performance using MV on parquet/orc.
+3. CDC and SCD.
```
Comment (jackylk) [inline README.md:77]: mention only for MERGE API for Spark DataFrame 
UPDATE and DELETE is production feature

Comment (kunal642) [reply README.md]: done

**Comment by jackylk** [comment]
LGTM

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.3.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.3/3117/

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.4.5, Please check CI http://121.244.95.60:12545/job/ApacheCarbon_PR_Builder_2.4.5/1393/

**Pull request closed without merging**